hw03_zapataa.py

```diff
@@ -0,0 +1,151 @@
+
+#################################################################################
+# Author: Adolfo Zapata
+# Username: zapataa
+#
+# Assignment: HW03
+# Purpose: To practice creating and using functions, as well as learning more about Git.
+# Google Doc Link:https://docs.google.com/document/d/15uYuTgh3gVOGoLn4trDeNj0eJUxllHVtLXcHX9tawog/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+#https://docs.python.org/3/library/turtle.html
+#
+#################################################################################
+
+import turtle
+
+def drawHouse(t):
```

Comment:
Pay attention to gray squigglies, they indicate style guidelines. This one says two blank lines before any function definitions.

Reply:
fixed syle guidlines